resources/assets/coffee/_classes/forum-topic-reply.coffee

```diff
@@ -115,6 +124,49 @@ class @ForumTopicReply
     @setState 'text', @input[0].value
 
 
+  fetchPreview: =>
+    return if $(@previewButton).hasClass('active')
+
+    if $(@previewButton).attr('data-should-refresh') == '0'
+      @showPreview()
+      return
+
+    $.ajax
+      url: $(@previewButton).attr('data-preview-url')
+      method: 'POST'
+      data:
+        body: $(@input).val()
+
+    .done (data) =>
+      $(@previewBox).html(data)
+
+      $(@previewButton).attr('data-should-refresh', '0')
```

Comment:
just store whatever text used in the element or wherever. See `PostAutoPreview` in `forum/post-box.coffee` (it needs to be moved as well).